docs/src/onlinereso.rst

```diff
@@ -1,7 +1,15 @@
 .. _librarywebsites:
 
-On-line resources
-=================
+Tutorials, recipes, and on-line resources
```

Comment:
Would it also make sense to add a link to MLIPs with i-PI? https://github.com/venkatkapil24/MLIPs-with-iPI